hibernate-core/src/main/java/org/hibernate/loader/entity/plan/LegacyBatchingEntityLoaderBuilder.java

```diff
@@ -47,6 +47,40 @@ protected UniqueEntityLoader buildBatchingLoader(
 		return new LegacyBatchingEntityLoader( persister, batchSize, lockOptions, factory, influencers );
 	}
 
+	@Override
+	protected UniqueEntityLoader buildBatchingLoader(
+			OuterJoinLoadable persister,
+			UniqueEntityLoader entityLoaderTemplate,
+			int batchSize,
+			LockMode lockMode,
+			SessionFactoryImplementor factory,
+			LoadQueryInfluencers influencers) {
+		if (entityLoaderTemplate instanceof LegacyBatchingEntityLoader) {
```

Comment:
As far as I know this violates the stupid code style used by Hibernate. You must add spaces after opening brace and before closing brace.

Reply:
Ah yes. I like this codestyle :). Missing the strict checkstyle rules we have in Search, OGM and Validator :). Will fix.